hnn_core/tests/test_optimization.py

```diff
@@ -90,21 +90,25 @@ def test_optimize_evoked():
     params.update({'N_pyr_x': 3,
                    'N_pyr_y': 3,
                    't_evprox_1': mu_orig,
-                   'sigma_t_evprox_1': 2.})
+                   'sigma_t_evprox_1': 2.,
+                   't_evdist_1': mu_orig + 2,
+                   'sigma_t_evdist_1': 2.})
     net_orig = jones_2009_model(params, add_drives_from_params=True)
     del net_orig.external_drives['evprox2']
-    del net_orig.external_drives['evdist1']
+    # del net_orig.external_drives['evdist1']
```

Comment:
don't leave commented code unless it's to explain something.

@rythorpe perhaps you want to take a look here since you wrote this originally?